README.md

```diff
@@ -1,5 +1,2 @@
-# swift-mongo-mobile
-
-## DISCLAIMER
-
-This driver is beta software. MongoDB SLAs do **NOT** apply to this beta software and it should **NOT** be used in production environments.
+# End of Life Notice
+MongoDB's swift-mongo-mobile is now officially end-of-life (EOL). No further development, bugfixes, enhancements, documentation changes or maintenance will be provided by this project and pull requests will no longer be accepted.
```

Comment:
might want to add a sentence directing people to Realm if they are looking to develop mobile applications using MongoDB
https://www.mongodb.com/realm

Reply:
This ^

Reply:
Added